meta-security-smack/lib/oeqa/runtime/smack.py

```diff
@@ -357,7 +357,7 @@ def test_smack_ambient(self):
 
         status, output = self.target.run("cat %s/ambient" %self.smack_path)
         # Filter '\x00', which is sometimes added to the ambient label
-        new_ambient = filter(lambda x: x in string.printable, output)
+        new_ambient = ''.join(list(filter(lambda x: x in string.printable, output)))
```

Comment:
Is that really necessary or did you just do a search/replace?

I understand the need in the other two cases (iterator object doesn't support random access), but here join() should be fine when applied to the filter() result.

Anyway, not terribly important.